src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    more requests can be read from the operating system buffer in a single
+    system call.
+
+    The recommendation is to make sure that the buffer can contain at least a
+    few dozen requests. Therefore, if a typical tuple in a request is large,
+    e.g. a few kilobytes or even megabytes, the read-ahead buffer size should
+    be increased. If batched request processing is not used, it's prudent to
+    leave this setting at its default.
+]])
+
+M['iproto.threads'] = format_text([[
+    The number of network threads. There can be unusual workloads where the
+    network thread is 100% loaded and the transaction processor thread is not,
+    so the network thread is a bottleneck. In that case, set `iproto_threads`
+    to 2 or more. The operating system kernel determines which connection goes
+    to which thread.
+]])
+
+-- }}} iproto configuration
+
+-- {{{ labels configuration
+
+M['labels'] = format_text([[
+    The `labels` section allows adding custom attributes to the configuration.
+    Attributes must be key: value pairs with string keys and values.
+]])
+
+M['labels.*'] = 'A value of the label with the specified name.'
+
+-- }}} labels configuration
+
+-- {{{ log configuration
+
+M['log'] = format_text([[
+    The `log` section defines configuration parameters related to logging.
+    To handle logging in your application, use the log module.
+]])
+
+M['log.file'] = format_text([[
+    Specify a file for logs destination. To write logs to a file, you need
+    to set `log.to` to file. Otherwise, `log.file` is ignored.
+]])
+
+M['log.format'] = format_text([[
+    Specify a format that is used for a log entry. The following formats
+    are supported:
+
+    - `plain`: a log entry is formatted as plain text.
+    - `json`: a log entry is formatted as JSON and includes additional fields.
+]])
+
+M['log.level'] = format_text([[
+    Specify the level of detail logs have. There are the following levels:
+
+    - 0: `fatal`
+    - 1: `syserror`
+    - 2: `error`
+    - 3: `crit`
+    - 4: `warn`
+    - 5: `info`
+    - 6: `verbose`
+    - 7: `debug`
+
+    By setting log.level, you can enable logging of all events with
+    severities above or equal to the given level.
+]])
+
+M['log.modules'] = format_text([[
+    Configure the specified log levels (`log.level`) for different modules.
+
+    You can specify a logging level for the following module types:
+
+    - Modules (files) that use the default logger. Example: Set log levels for
+    files that use the default logger.
+    - Modules that use custom loggers created using the `log.new()` function.
+    Example: Set log levels for modules that use custom loggers.
+    - The tarantool module that enables you to configure the logging level
+    for Tarantool core messages. Specifically, it configures the logging level
+    for messages logged from non-Lua code, including C modules. Example: Set
+    a log level for C modules.
+]])
```

Comment:
`Example: <...>` has no sense without a link. I would drop it until we have links support.

Reply:
Dropped.

Reply:
> These links were in each list item and the first two still contain it.

oops.. Dropped.
```lua
I['log.modules'] = format_text([[
    Configure the specified log levels (`log.level`) for different modules.

    You can specify a logging level for the following module types:

    - Modules (files) that use the default logger.
    - Modules that use custom loggers created using the `log.new()` function.
    - The tarantool module that enables you to configure the logging level
      for Tarantool core messages. Specifically, it configures the logging level
      for messages logged from non-Lua code, including C modules.
]])
```